#goal
Header header
geometry_msgs/Twist targetTwist
geometry_msgs/Pose targetPose
bool stationKeeping
---
#result
bool Ok
---
#feedback
float32 distanceError
float32 rotationError
float32 velocityError
float32 angularVelError
